src/Executables/ReduceCceWorldtube/ReduceCceWorldtube.cpp

```diff
@@ -152,10 +152,10 @@ void slice_buffers_to_libsharp_modes(
 // read in the data from a (previously standard) SpEC worldtube file
 // `input_file`, perform the boundary computation, and dump the (considerably
 // smaller) dataset associated with the spin-weighted scalars to `output_file`.
-void perform_cce_worldtube_reduction(const std::string& input_file,
-                                     const std::string& output_file,
-                                     const size_t buffer_depth,
-                                     const size_t l_max_factor) noexcept {
+void perform_cce_worldtube_reduction(
+    const std::string& input_file, const std::string& output_file,
+    const size_t buffer_depth, const size_t l_max_factor,
+    const bool apply_spec_corrections = false) noexcept {
```

Comment:
Is this conceptually the same switch as the `h5_spec_normalization` in the other function? It's not obvious to me why it has a different name.

Reply:
It's the same use -- I'll change this one to be consistent.